src/article_metrics/api_v2_views.py

```diff
@@ -19,6 +19,13 @@
 LOG = logging.getLogger(__name__)
 
 PROFILING = False
+
+ctype_idx = {
```

Comment:
minor: maybe could now use upper case letters since it became a constant (could also do with a better name but this is what we had before)